presto-client/pom.xml

```diff
@@ -87,5 +87,9 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Can you move this up (so it's not next to the test dependencies)?

Reply:
done